src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -71,25 +73,23 @@ public void updateTaskForStatus(Protos.TaskStatus status) {
       // If initializing the first NN or reconciling the first NN or bootstrapping the first NN
       // set the status to initialized
       if (status.getMessage().equals(HDFSConstants.NAME_NODE_INIT_MESSAGE)
```

Comment:
Hm, sometimes parans can add clarity and readability, although there's no logical change. I prefer either removing the useless parans assuming it doesn't impact the readability and not having the NOPMD comment